atuin-server/src/handlers/history.rs

```diff
@@ -140,8 +144,24 @@ pub async fn add<DB: Database>(
         })
         .collect();
 
-    let db = &state.0.database;
-    if let Err(e) = db.add_history(&history).await {
+    history.retain(|h| {
```

Comment:
This is changing the behaviour right? So if someone inserts something too large, their client will continue trying to sync

Reply:
Shouldn't be any change in functionality, I just move it from the database layer to the application layer https://github.com/ellie/atuin/blob/main/atuin-server/src/database.rs#L305-L318

Reply:
Ahhh of course! Makes sense, thanks